docs/os/build-details.md

```diff
@@ -15,7 +15,7 @@ It performs the following steps:
 1. cloning the [`github.com/armbian/build`](https://github.com/armbian/build/) repo into `armbian/armbian-build/`, if it doesn't exist already
 1. copying over the following files from the host system into `armbian/armbian-build/userpatches/overlay/`:
     1. `armbian/base/`: scripts and configs included in the Armbian image
-    1. `build/`: contains all built Go binaries and their associated `.service` files
+    1. `build/`: contains all [built Go binaries](/go/) and their associated `.service` files
```

Comment:
I think that should be `/docs/go-services.md`

Reply:
Ah, I was trying to guess / reverse-engineer what URLs Jekyll would render these pages on (that was the reason for adding `permalink` stanzas too, btw), but if we can refer to the `.md` files and let Jekyll sort out what `<a href="...">` values to put in, that's certainly better!

Fixed, thanks.